_posts/2025-02-04-introduce-bitmap-filtering-feature.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title:  "Efficient large-scale filtering with bitmap filtering in OpenSearch"
+authors:
+   - bowenlan-amzn
+   - macrakis
+   - msfroh
+   - kolchfa
+date: 2025-02-04
+categories:
+  - technical-posts
+meta_keywords: bitmap
+meta_description: Introduce the bitmap filtering feature about its usage and performance
```

Comment:
Update meta with the following: 
_meta_keywords:_ bitmap filtering, OpenSearch 2.17, filtering large datasets, RoaringBitmap, OpenSearch queries, e-commerce search. search performance

_meta_description:_ Discover how the bitmap filtering feature in OpenSearch optimizes large-scale filtering operations, and improves query performance and efficiency for datasets with thousands to millions of terms